assets/airports/rjaa.json

```diff
@@ -0,0 +1,1153 @@
+{
+    "radio": {
+        "twr": "Narita Tower",
+        "app": "Tokyo Approach",
+        "dep": "Tokyo Departure"
+    },
+    "icao": "RJAA",
+    "iata": "NRT",
+    "magnetic_north": 0,
+    "ctr_radius": 80,
+    "ctr_ceiling": 27000,
```

Comment:
Also note that this is still taken in hundreds of feet, so you would want it to take the form of `270`.